lib/std/packedsets.nim

```diff
@@ -541,14 +543,12 @@ proc `-`*[A](s1, s2: PackedSet[A]): PackedSet[A] {.inline.} =
 proc disjoint*[A](s1, s2: PackedSet[A]): bool =
   ## Returns true if the sets `s1` and `s2` have no items in common.
   runnableExamples:
-    var
-      a = initPackedSet[int]()
-      b = initPackedSet[int]()
-    a.incl(1); a.incl(2)
-    b.incl(2); b.incl(3)
+    let
+      a = toPackedSet([1, 2])
```

Comment:
optional nit: prefer MCS/UFCS form everywhere, which is simpler: `a = [1, 2].toPackedSet`